fusor-ember-cli/app/templates/review/summary.hbs

```diff
@@ -36,6 +36,14 @@
         {{/accordion-item}}
     {{/if}}
 
+    {{#if isOpenShift}}
+        {{#accordion-item name="Red Hat Enterprise Linux - OpenShift Enterprise" isOpen=isOpenShiftOpen}}
```

Comment:
+1 to just "OpenShift Enterprise"   (dropping the "Red Hat Enterprise Linux -") 


@jmrodri @dymurray  I think it's "OpenShift Enterprise" and not "Openshift Enterprise" based on below:

https://www.openshift.com/enterprise

"OpenShift Enterprise, an award-winning private PaaS, uses containers to help you speed innovation and deliver apps faster."